test-utils/examples/generate_testnet_configuration.rs

```diff
@@ -12,12 +12,16 @@ use templar_common::{
     market::{BalanceOracleConfiguration, MarketConfiguration, YieldWeights},
     number::Decimal,
     oracle::pyth::PriceIdentifier,
+    time_chunk::TimeChunkConfiguration,
 };
 
 pub fn main() {
     println!(
         "{{\"configuration\":{}}}",
         serde_json::to_string(&MarketConfiguration {
+            time_chunk_configuration: TimeChunkConfiguration::BlockTimestampMs {
+                divisor: (1000u64 * 60 * 10).into(), // every 10 minutes
```

Comment:
Small readabitliy nit:
```suggestion
                divisor: (1_000u64 * 60 * 10).into(), // every 10 minutes
```

Reply:
I will add stricter Clippy rules to detect+fix this.